docs/contributing/building.rst

```diff
@@ -4,7 +4,7 @@
 Building
 ========
 
-If you are building katana-enterprise make sure to also read the `enterprise build addenda <enterprise-building>`_.
+If you are building katana-enterprise make sure to also read the :doc:`enterprise build addenda <index>`.
```

Comment:
Yea. I noticed this. Thanks. Sorry about this.

Could we link directly to the enterprise build section? It seems like it should be possible in RST.

Reply:
We have sphinx configured to promote warnings to errors, and now that this is an internal link, Sphinx knows that it is dangling in the open documentation.